src/main/resources/templates/fragments/inputs/select.html

```diff
@@ -4,6 +4,7 @@
       hasHelpText=${!#strings.isEmpty(helpText)},
       hasLabel=${!#strings.isEmpty(label)},
       hasAriaLabel=${!#strings.isEmpty(ariaLabel)},
+      isRequiredInput=${requiredInputs.getOrDefault(inputName, false) || required != null && required},
```

Comment:
I would add parenthesis around the `required` part, like so `(required != null && required)`.

Reply:
Done